ui/v2.5/src/components/Studios/StudioCard.tsx

```diff
@@ -51,14 +51,16 @@ function maybeRenderParent(
 
 function maybeRenderChildren(studio: GQL.StudioDataFragment) {
   if (studio.child_studios.length > 0) {
+    const studioLabel =
+      studio.child_studios.length === 1 ? "studio" : "studios";
```

Comment:
This needs to be internationalised. Take a look how it's done in [TagCard](https://github.com/stashapp/stash/blob/c74456c07e1cce1140cce8633882c31b521c2e9b/ui/v2.5/src/components/Tags/TagCard.tsx#L123)